### PR TITLE
Index less

### PR DESCRIPTION
**Pull request by stevemessick**
@pq @devoncarew 

For Flutter projects created with Android Studio, project creation should be quiet:
- Android framework is not detected, so there is no notification to configure it
- Constant 2-sec refresh is disabled
- "Migrate to Gradle" is not suggested

Opening a Flutter project created in IntelliJ:
- Constant 2-sec refresh is disabled
- "Migrate to Gradle" is suggested only first time the first time a project is opened -- `FlutterUtils.disableGradleProjectMigrationNotification()` is not run until after the notification is displayed, but does work for subsequent openings

That last item needs further work. The user won't see the prompt twice, even if no action is taken, but they should not even see it once. I think it is a problem that needs to be solved in IntelliJ.

### CODE REVIEW

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -615,7 +615,7 @@ miscellaneous:
     <projectService serviceInterface="io.flutter.view.FlutterView"
                     serviceImplementation="io.flutter.view.FlutterView"
                     overrides="false"/>
-    <projectOpenProcessor implementation="io.flutter.project.FlutterProjectOpenProcessor" order="first"/>
+    <projectOpenProcessor id="flutter" implementation="io.flutter.project.FlutterProjectOpenProcessor" order="first"/>
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: Is this id unique enough? I'm not sure what other ids in this space would look like.

File: resources/META-INF/studio-contribs.xml [hunk 2]
```diff
@@ -15,6 +15,7 @@
   <extensions defaultExtensionNs="com.intellij">
     <androidStudioInitializer implementation="io.flutter.FlutterStudioInitializer"/>
     <postStartupActivity implementation="io.flutter.FlutterStudioStartupActivity"/>
+    <projectOpenProcessor implementation="io.flutter.project.FlutterStudioProjectOpenProcessor" order="after flutter"/>
```
Comment (devoncarew) [inline resources/META-INF/studio-contribs.xml]: It's ok for these project open processors to not have ids?

File: resources/META-INF/plugin.xml (hunk 1 above)
Comment (stevemessick) [reply resources/META-INF/plugin.xml]: None of the others have id's, so I think this is good. It is a very small name-space

File: resources/META-INF/studio-contribs.xml (hunk 2 above)
Comment (stevemessick) [reply resources/META-INF/studio-contribs.xml]: Hardly any extensions have id's. I should probably add one, just in case.

**Pull request merged**